docs/index.rst

```diff
@@ -57,6 +57,7 @@ Rockcraft is a member of the Canonical family. It's an open source project
 that warmly welcomes community projects, contributions, suggestions, fixes
 and constructive feedback.
 
+* `Ubuntu ROCKS discourse <https://discourse.ubuntu.com/c/rocks/117>`_
```

Comment:
```suggestion
* `Ubuntu ROCKs Discourse <https://discourse.ubuntu.com/c/rocks/117>`_
```

Reply:
Unresolving as the change wasn't fully applied. It's normally safer to directly commit easy suggestions like these, to avoid missing nuanced changes when trying to reproduce the change request. The commit squashing can be done later.

Reply:
It is really now on the branch, but I resolved before committing hence this behavior.

Reply:
You missing the "S" vs "s" :)

Reply:
Opps, my mistake!